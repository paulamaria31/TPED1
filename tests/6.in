250
XVXVXOVVV
VVOXVOVOV
XOXVXVVXV
VVXOXXOOO
XVOVOXOVX
XVOXOOVXX
VOOXXXVVO
XXVOOXOXO
OOVXVOOVV
VVVOOXXXX
VVVOXXVOV
XVXVOOXOX
VXVVVVOOX
OVXXVXXOX
OOOXOOOOV
OOOVXXOXV
XOXVXOXXV
VXOOXOXXX
VOXOVVOOX
VOVOOVVOX
XOXXVXXXX
VOVXOXOOX
VXXVVOVOX
OVOOOOVXV
OVVXXOXVV
OOOXOOVXX
VXVOXVOOO
OOVXOXOOX
VXXXVXVOV
VOOVVOOVO
OOXXOOVXX
VVOVOXOVO
OVOVXXOXX
OVXVOOOVV
OXVXVXOOX
XOXVVXVOV
OOOXOVXXX
OXVXOVOVV
VOVVXXXXV
XXVVXOXXX
XXVVVXXVV
VXXXXOVOX
XVOVXVVVO
XVVXXVVXO
OVXOXOOVX
XXXOVOOXX
VVXOVOOVX
OOOVOVXVX
VVVVXOXVX
XOOOOOXXO
OVVOOOOVV
VVXVOVVXV
VOOXOVOXX
VXOVVOVOO
VVVVXVXOO
VXXOVXVXX
OXXXVXXVX
XVOVVVOVO
XVOXXVVVO
VVXOOVXOV
XVOXOOXXV
VOVXOVVOO
VXOVOXOVV
VVVXVXXXO
VXVXXVVVV
VOXXXVOOX
OOOOVXXVX
XXXVXVVVV
OOVOOXOXX
OVOVVVOVX
OOOVXVOVX
OOVXVVOVO
VXXXOOVXX
OXOVXOOXX
OVVVOXVXX
VVVXVOOXX
OVXXOOVXV
VXXOOVXOO
XXXXOOOVO
XVOVVOVOO
OXOVOVXVX
XVVXXOXVV
VXVXVXOVO
XVXOOXVVO
VVXVXXOOO
XOXOXXXXX
VXXVXXOVO
XOVOXXVVV
XXXOXVVOO
OXXVXVOXV
OVOXOXVVV
OOXXXOXOO
VXVVOOXOX
OOOOOVOXX
XVVOOXXVO
VXVVVVVXO
XOXVVVVVO
VXXXOVOOX
XXXVOXOXV
OVVOOVVOO
XVOXXOXXX
XVVXVXXOO
OOOOOOOOX
VXXVVOVOV
VXXVXVVOO
VXVOOXOVV
XXVVOVXOV
OXVOOOOOO
XVXOOXVVO
VXOVVVXXO
XVVVXOXOV
XXVVVXVOO
VOOXXVOXO
OXXXOOXVO
OXXXXXVVO
OXOOOXXOX
VVOVXVVXX
XXXVXVOOV
XXXXXOOVX
OXOXVXOVX
XXXOVOXVO
VVOXVXVVV
OOVVOVOXX
XXOXXXVOO
XVVXXOOOO
XVOVVXVXX
OVOOXVVXO
VVOXOXOVV
OVXOOVXVV
VVVOXXXVX
XVXXOXXVO
OXXVVXXOO
OXXOOOXXX
VVOXOOVVX
OXOVXVVXO
OVVOOVOOO
OOOOVOXOO
XXXVXVVOO
XVOVXVOOO
OOVOOVVOV
VVXXVVXVV
XXVXVXOOV
XXVVXXXXO
OVXXOOVOX
VVVXOVVXX
XVOXVVVVO
OVXOXOOVX
VOXXOVVVO
OOOXVXVOV
VOVXVVXVX
VXVOXOXVO
OVXXOOXOV
XOXOXOXVX
VXVVVVOVX
OXXVXVVVV
VVXVOOXOX
VOVOOOOXX
OOXOVOOOO
XVXXXVVOV
VVVOOOOVX
XOOXVXOOV
VOVOXOXOO
OOOVXXVVX
XXOXXOOVO
VXVVOXOVX
OXXVVVOXX
XXVOVVOVV
XXOVXVOVV
XVOVVXOXV
VOOOVOVOO
XXVVXOVXO
XVOXOVXOV
OVVVXOOXV
OVOVXOOXO
XVXOOXVOX
XXVVVOXVX
VXOOXVXXV
XOVVOXOVO
VOVXVXOOO
OOXVXXOVO
XOVXOOOOX
OOXXXOOXO
OOOXOVVOO
VXOVOVVVV
XOXXVVOVX
XVXVOVVXO
VOVVOXVXV
VXXOVVVXO
VOOXVVVOX
OVVOXVVOO
OVVVOXVXO
XVXVOVOXO
VOOXVXXOV
VOVXOOOOX
XVOVXOVXV
VOVVVXXOV
XXXVVVVXO
XVVOXVXXV
OXVVXVVXV
OXOXVVXVV
VXOXXVOOV
OXOOVVOOO
VOOVXOVVV
OOVXOXXOX
OXVXOVXVV
XOOVVVVOO
XVOVOVOXV
VOVVXXOVV
VOXXOXVXV
OVXVXOVOX
VXOOOXVOO
XXOVXXXOX
OOOOOOVOO
VVOXXOVOV
XOVVOVXVO
VVVXXVXXX
VOXXXVVOX
OOVXVVVVX
VVVXOXXOV
XOVVVVXVV
VVOXXOVVV
XOOOXXXVO
OOOOXXVXO
VOVOOXOOX
VOVVOVOVX
VVXXVOXOV
XVVVXVXVX
XOXXOXOVV
OXXVXOOVO
OVVXVXOOX
XOVOXXOXX
XXVOXOVXX
XXXXVOXXX
OOXOXXOOO
XVXVXOVOX
XVXVVXXOO
VVXXXOVXX
VXXVXXOXV
VVVOXOXVX
VXVVOOVXO
OOXXXXXOX
VXXXXOOVX
OOXVOOXVX
XVOOOOXXO
XXOOOVXOX
XXOXXVVXO
OXVXXOXXO
OXOOVXVXV
OOVOOVOOX
VOVOOVOVX
