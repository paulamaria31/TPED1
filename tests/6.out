Tabuleiro 1 invalido
Tabuleiro 2 invalido
Tabuleiro 3 invalido
Tabuleiro 4 com vitoria [O]
Tabuleiro 5 com vitoria [O]
Tabuleiro 6 em andamento [O: (3,1)]
Tabuleiro 7 com vitoria [X]
Tabuleiro 8 em andamento [proximo jogador indefinido]
Tabuleiro 9 invalido
Tabuleiro 10 invalido
Tabuleiro 11 em andamento [proximo jogador indefinido]
Tabuleiro 12 em andamento [O: (2,1)(1,2)]
Tabuleiro 13 em andamento [proximo jogador indefinido]
Tabuleiro 14 invalido
Tabuleiro 15 invalido
Tabuleiro 16 com vitoria [O]
Tabuleiro 17 invalido
Tabuleiro 18 invalido
Tabuleiro 19 invalido
Tabuleiro 20 invalido
Tabuleiro 21 invalido
Tabuleiro 22 com vitoria [O]
Tabuleiro 23 em andamento [O: sem jogada mestre]
Tabuleiro 24 invalido
Tabuleiro 25 em andamento [O: sem jogada mestre]
Tabuleiro 26 invalido
Tabuleiro 27 invalido
Tabuleiro 28 invalido
Tabuleiro 29 invalido
Tabuleiro 30 invalido
Tabuleiro 31 em andamento [proximo jogador indefinido]
Tabuleiro 32 invalido
Tabuleiro 33 em andamento [O: (1,2)(2,1)]
Tabuleiro 34 invalido
Tabuleiro 35 em andamento [O: sem jogada mestre]
Tabuleiro 36 em andamento [O: (2,2)]
Tabuleiro 37 invalido
Tabuleiro 38 em andamento [X: sem jogada mestre]
Tabuleiro 39 invalido
Tabuleiro 40 invalido
Tabuleiro 41 invalido
Tabuleiro 42 invalido
Tabuleiro 43 em andamento [proximo jogador indefinido]
Tabuleiro 44 invalido
Tabuleiro 45 com vitoria [O]
Tabuleiro 46 invalido
Tabuleiro 47 em andamento [X: sem jogada mestre]
Tabuleiro 48 invalido
Tabuleiro 49 invalido
Tabuleiro 50 invalido
Tabuleiro 51 invalido
Tabuleiro 52 em andamento [O: sem jogada mestre]
Tabuleiro 53 com vitoria [O]
Tabuleiro 54 invalido
Tabuleiro 55 em andamento [proximo jogador indefinido]
Tabuleiro 56 invalido
Tabuleiro 57 invalido
Tabuleiro 58 invalido
Tabuleiro 59 em andamento [O: (2,3)]
Tabuleiro 60 em andamento [X: sem jogada mestre]
Tabuleiro 61 com vitoria [X]
Tabuleiro 62 invalido
Tabuleiro 63 com vitoria [O]
Tabuleiro 64 invalido
Tabuleiro 65 invalido
Tabuleiro 66 em andamento [O: sem jogada mestre]
Tabuleiro 67 com vitoria [O]
Tabuleiro 68 invalido
Tabuleiro 69 invalido
Tabuleiro 70 invalido
Tabuleiro 71 invalido
Tabuleiro 72 invalido
Tabuleiro 73 invalido
Tabuleiro 74 com vitoria [X]
Tabuleiro 75 em andamento [O: sem jogada mestre]
Tabuleiro 76 em andamento [O: sem jogada mestre]
Tabuleiro 77 em andamento [proximo jogador indefinido]
Tabuleiro 78 em andamento [X: (1,1)]
Tabuleiro 79 com vitoria [X]
Tabuleiro 80 invalido
Tabuleiro 81 em andamento [proximo jogador indefinido]
Tabuleiro 82 invalido
Tabuleiro 83 em andamento [O: (3,2)]
Tabuleiro 84 em andamento [proximo jogador indefinido]
Tabuleiro 85 com vitoria [O]
Tabuleiro 86 invalido
Tabuleiro 87 invalido
Tabuleiro 88 em andamento [O: sem jogada mestre]
Tabuleiro 89 com vitoria [X]
Tabuleiro 90 invalido
Tabuleiro 91 em andamento [X: sem jogada mestre]
Tabuleiro 92 com vitoria [X]
Tabuleiro 93 em andamento [proximo jogador indefinido]
Tabuleiro 94 invalido
Tabuleiro 95 em andamento [proximo jogador indefinido]
Tabuleiro 96 em andamento [O: sem jogada mestre]
Tabuleiro 97 em andamento [proximo jogador indefinido]
Tabuleiro 98 em andamento [O: sem jogada mestre]
Tabuleiro 99 invalido
Tabuleiro 100 invalido
Tabuleiro 101 invalido
Tabuleiro 102 invalido
Tabuleiro 103 invalido
Tabuleiro 104 em andamento [proximo jogador indefinido]
Tabuleiro 105 em andamento [O: (3,1)]
Tabuleiro 106 em andamento [X: sem jogada mestre]
Tabuleiro 107 em andamento [O: sem jogada mestre]
Tabuleiro 108 invalido
Tabuleiro 109 em andamento [proximo jogador indefinido]
Tabuleiro 110 em andamento [O: (2,3)]
Tabuleiro 111 em andamento [O: sem jogada mestre]
Tabuleiro 112 em andamento [O: (3,1)]
Tabuleiro 113 em andamento [X: (2,3)]
Tabuleiro 114 com vitoria [O]
Tabuleiro 115 invalido
Tabuleiro 116 deu velha
Tabuleiro 117 invalido
Tabuleiro 118 invalido
Tabuleiro 119 invalido
Tabuleiro 120 em andamento [O: (2,2)]
Tabuleiro 121 com vitoria [X]
Tabuleiro 122 em andamento [O: sem jogada mestre]
Tabuleiro 123 invalido
Tabuleiro 124 invalido
Tabuleiro 125 com vitoria [O]
Tabuleiro 126 invalido
Tabuleiro 127 invalido
Tabuleiro 128 com vitoria [O]
Tabuleiro 129 em andamento [X: sem jogada mestre]
Tabuleiro 130 invalido
Tabuleiro 131 invalido
Tabuleiro 132 em andamento [O: (2,2)]
Tabuleiro 133 invalido
Tabuleiro 134 em andamento [X: sem jogada mestre]
Tabuleiro 135 com vitoria [X]
Tabuleiro 136 invalido
Tabuleiro 137 invalido
Tabuleiro 138 invalido
Tabuleiro 139 invalido
Tabuleiro 140 invalido
Tabuleiro 141 invalido
Tabuleiro 142 invalido
Tabuleiro 143 invalido
Tabuleiro 144 em andamento [X: sem jogada mestre]
Tabuleiro 145 invalido
Tabuleiro 146 em andamento [proximo jogador indefinido]
Tabuleiro 147 com vitoria [O]
Tabuleiro 148 em andamento [X: sem jogada mestre]
Tabuleiro 149 invalido
Tabuleiro 150 invalido
Tabuleiro 151 em andamento [proximo jogador indefinido]
Tabuleiro 152 em andamento [X: sem jogada mestre]
Tabuleiro 153 invalido
Tabuleiro 154 em andamento [O: sem jogada mestre]
Tabuleiro 155 invalido
Tabuleiro 156 em andamento [proximo jogador indefinido]
Tabuleiro 157 invalido
Tabuleiro 158 invalido
Tabuleiro 159 invalido
Tabuleiro 160 invalido
Tabuleiro 161 em andamento [X: (2,2)]
Tabuleiro 162 invalido
Tabuleiro 163 com vitoria [O]
Tabuleiro 164 com vitoria [O]
Tabuleiro 165 em andamento [O: (1,3)]
Tabuleiro 166 invalido
Tabuleiro 167 em andamento [proximo jogador indefinido]
Tabuleiro 168 em andamento [O: sem jogada mestre]
Tabuleiro 169 em andamento [O: (2,2)]
Tabuleiro 170 invalido
Tabuleiro 171 invalido
Tabuleiro 172 com vitoria [X]
Tabuleiro 173 em andamento [X: (1,2)]
Tabuleiro 174 invalido
Tabuleiro 175 com vitoria [X]
Tabuleiro 176 invalido
Tabuleiro 177 invalido
Tabuleiro 178 invalido
Tabuleiro 179 invalido
Tabuleiro 180 em andamento [X: (2,1)]
Tabuleiro 181 invalido
Tabuleiro 182 deu velha
Tabuleiro 183 invalido
Tabuleiro 184 em andamento [X: sem jogada mestre]
Tabuleiro 185 invalido
Tabuleiro 186 em andamento [O: sem jogada mestre]
Tabuleiro 187 em andamento [proximo jogador indefinido]
Tabuleiro 188 em andamento [O: sem jogada mestre]
Tabuleiro 189 em andamento [X: sem jogada mestre]
Tabuleiro 190 invalido
Tabuleiro 191 com vitoria [O]
Tabuleiro 192 em andamento [proximo jogador indefinido]
Tabuleiro 193 em andamento [proximo jogador indefinido]
Tabuleiro 194 invalido
Tabuleiro 195 em andamento [O: (3,3)]
Tabuleiro 196 em andamento [proximo jogador indefinido]
Tabuleiro 197 invalido
Tabuleiro 198 invalido
Tabuleiro 199 invalido
Tabuleiro 200 em andamento [O: sem jogada mestre]
Tabuleiro 201 em andamento [proximo jogador indefinido]
Tabuleiro 202 invalido
Tabuleiro 203 invalido
Tabuleiro 204 com vitoria [O]
Tabuleiro 205 em andamento [O: (3,3)]
Tabuleiro 206 invalido
Tabuleiro 207 com vitoria [O]
Tabuleiro 208 em andamento [proximo jogador indefinido]
Tabuleiro 209 invalido
Tabuleiro 210 em andamento [proximo jogador indefinido]
Tabuleiro 211 invalido
Tabuleiro 212 invalido
Tabuleiro 213 invalido
Tabuleiro 214 em andamento [X: sem jogada mestre]
Tabuleiro 215 em andamento [X: (2,1)]
Tabuleiro 216 invalido
Tabuleiro 217 invalido
Tabuleiro 218 em andamento [proximo jogador indefinido]
Tabuleiro 219 em andamento [O: (1,2)]
Tabuleiro 220 em andamento [O: sem jogada mestre]
Tabuleiro 221 em andamento [proximo jogador indefinido]
Tabuleiro 222 em andamento [proximo jogador indefinido]
Tabuleiro 223 invalido
Tabuleiro 224 invalido
Tabuleiro 225 invalido
Tabuleiro 226 em andamento [O: sem jogada mestre]
Tabuleiro 227 invalido
Tabuleiro 228 em andamento [O: (3,2)]
Tabuleiro 229 em andamento [X: (3,2)]
Tabuleiro 230 em andamento [proximo jogador indefinido]
Tabuleiro 231 invalido
Tabuleiro 232 invalido
Tabuleiro 233 invalido
Tabuleiro 234 invalido
Tabuleiro 235 invalido
Tabuleiro 236 invalido
Tabuleiro 237 invalido
Tabuleiro 238 invalido
Tabuleiro 239 em andamento [O: sem jogada mestre]
Tabuleiro 240 em andamento [X: sem jogada mestre]
Tabuleiro 241 invalido
Tabuleiro 242 invalido
Tabuleiro 243 em andamento [X: (3,2)]
Tabuleiro 244 invalido
Tabuleiro 245 em andamento [proximo jogador indefinido]
Tabuleiro 246 invalido
Tabuleiro 247 invalido
Tabuleiro 248 em andamento [proximo jogador indefinido]
Tabuleiro 249 invalido
Tabuleiro 250 invalido
